modules/pel/peltool/peltool.py

```diff
@@ -267,6 +267,36 @@ def parseAndWriteOutput(file: str, output_dir: str, config: Config,
             print(f"No PEL parsed for {file}: {e}")
 
 
+def parsePelID(path: str, pelID: str, config: Config) -> None:
```

Comment:
This function name kinda sounds like its just the PEL ID being parsed.  How about something like parsePelFromId or something along those lines.